Examples/Run/Vertexing/TrackReaderVertexingPerformanceWriterExample.cpp

```diff
@@ -61,11 +61,10 @@ int main(int argc, char* argv[]) {
 
   RootParticleReader::Config particleReaderConfig;
   particleReaderConfig.particleCollection = "allTruthParticles";
-  particleReaderConfig.inputFile = "particles_initial.root";
+  particleReaderConfig.filePath = inputDir + "/particles_initial.root";
```

Comment:
As I said in #915 I think inputFilePath would look better. But I leave that up to you.